lib/compat/plugin/footnotes.php

```diff
@@ -0,0 +1,250 @@
+<?php
```

Comment:
@ellatrix If I'm not wrong this file should be moved to `lib/compat/wordpress-6.4` because it's just here to support versions priori to 6.4 right?

Reply:
Yes, although maybe I don't think these make sense anymore after the revision support was added. Cc @adamsilverstein